### PR TITLE
mandatory update screen

### PR DESCRIPTION
**Pull request by vadimpylsky**

### CODE REVIEW

File: src/lib/services/auth_service.dart [hunk 1]
```diff
@@ -59,6 +59,6 @@ class AuthService extends BaseAuthService {
   }
 
   bool _validateEmail(String email) {
-    return email.endsWith("@softeq.com");
+    return true;
```
Comment (wcoder) [inline src/lib/services/auth_service.dart:62]: Please revert check

File: src/lib/viewmodels/auth_viewmodel.dart [hunk 2]
```diff
@@ -1,3 +1,5 @@
+import 'dart:async';
+
```
Comment (wcoder) [inline src/lib/viewmodels/auth_viewmodel.dart:2]: Rudimental import

File: src/lib/pages/mandatory_update_page.dart [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+              ),
+              SizedBox(height: 16.0),
+              Text(
+                localizer().updateRequired,
+                style: KudosTheme.listTitleTextStyle,
+              ),
+              SizedBox(height: 16.0),
+              Text(
+                localizer().updateRequiredMessage,
+                textAlign: TextAlign.center,
```
Comment (wcoder) [inline src/lib/pages/mandatory_update_page.dart:28]: Optional: maybe add a button to open the appcener?

File: src/lib/kudos_app.dart [hunk 4]
```diff
@@ -61,12 +63,14 @@ class KudosApp extends StatelessWidget {
...
         _analyticsService.observer,
       ],
+      navigatorKey: locator<NavigationService>().navigatorKey,
     );
   }
 
   Widget _buildHome(BuildContext context, AuthViewModel viewModel) {
     switch (viewModel.authState) {
       case AuthViewModelState.loggedIn:
+        locator<MandatoryUpdateDatabaseService>().refreshSubscriptions();
```
Comment (wcoder) [inline src/lib/kudos_app.dart:73]: Can be better to move in `HomeViewModel`

**Review by wcoder — CHANGES_REQUESTED** [review]

File: src/lib/viewmodels/auth_viewmodel.dart (hunk 2 above)
Comment (vadimpylsky) [reply src/lib/viewmodels/auth_viewmodel.dart:2]: resolved

File: src/lib/services/auth_service.dart (hunk 1 above)
Comment (vadimpylsky) [reply src/lib/services/auth_service.dart:62]: resolved

**Review by wcoder — APPROVED** [review]

**Review by wcoder — APPROVED** [review]

**Pull request merged**